Gemfile

```diff
@@ -1,5 +1,5 @@
-source 'http://rubygems.org'
-ruby '~>2.6.5'
+source 'https://rubygems.org'
+ruby '~>3.2.2'
 
 gem 'pry-byebug'
```

Comment:
Not just a developer dependency, I think; it also "ties" this version to this ruby for clients.  So if there is a client out there using Ruby 2.6 still, they won't be able to access this new gem because their ruby version is compatible.

Might be more academic than anything, since I don't think there are many external users of this gem, though :)